go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/smartcontractkit/chainlink/v2
 
-go 1.21.3
+go 1.21.5
```

Comment:
Any reason this is a different vers than `core/scripts/go.mod`?

Reply:
good question - @jmank88 are all of them now `1.21.7`?